hack/testdata/rollingupdate-statefulset.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1beta1
```

Comment:
> Would you change the rollingupdate-daemonset yaml files (under testdata) to `apps/v1beta2` as well while you're at it?

Actually, we can do this separately in another PR. I'll file a PR for this, you just need to modify the statefulset manifest to use `apps/v1beta2`